Composer/packages/client/src/pages/diagnostics/Diagnostics.tsx

```diff
@@ -55,7 +58,7 @@ const Diagnostics: React.FC<RouteComponentProps<{ projectId: string; skillId: st
       toolbarItems={toolbarItems}
       onRenderHeaderContent={onRenderHeaderContent}
     >
-      <DiagnosticList showType={showType} skillId={skillId ?? projectId} onItemClick={handleItemClick} />
+      <DiagnosticsTable showType={showType} skillId={skillId ?? projectId} onItemClick={handleItemClick} />
```

Comment:
Is the reason we are updating the Diagnostics page is to add Pagination support? Any other functionalities added to that page?

Reply:
The motivation is to take out pagination logic out of <DiagnosticList /> and reuse it in Debug Panel.
Pagination + DiagnosticList are composited as Diagnosticstable.